assets/editorial_features.styl

```diff
@@ -1,3 +1,3 @@
 @require '../components/main_layout/stylesheets'
 @require '../components/article/stylesheets'
-@require '../apps/editorial_features/components/eoy'
+@require '../apps/editorial_features/components/eoy/stylesheets/eoy'
```

Comment:
I think this should be `../apps/editorial_features/components/eoy/stylesheets`, right? Based on the stylesheets you have: `apps/editorial_features/components/eoy/stylesheets/scroller.styl`